Sources/OpenCombine/Published.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  Published.swift
+//  OpenCombine
+//
+//  Created by Евгений Богомолов on 01/09/2019.
+//
+
+import Foundation
+
+@propertyWrapper public struct Published<Value> {
```

Comment:
Let's wrap this whole file in conditional compilation block to be compatible with Swift 5:

```swift
#if swift(>=5.1)
...
#endif
```

Reply:
Please don't mark the conversation as resolved until you commit the changes